tagstudio/src/core/palette.py

```diff
@@ -1,274 +1,281 @@
 # Copyright (C) 2024 Travis Abendshien (CyanVoxel).
 # Licensed under the GPL-3.0 License.
 # Created for TagStudio: https://github.com/CyanVoxel/TagStudio
+import traceback
+from enum import IntEnum
+from typing import Any
 
-from enum import Enum
+import structlog
 
+from src.core.library.alchemy.enums import TagColor
 
-class ColorType(int, Enum):
+logger = structlog.get_logger(__name__)
```

Comment:
for now I just wanted to focus on the actual database than niceties around. Using `logging` is cumbersome so I replaced it with `structlog` but I dont want to think where and how to have logging placed and configured, so I use this approach for now. This can sure be refactored out later.